lib/charms/mongodb/v0/mongodb_backups.py

```diff
@@ -272,9 +271,11 @@ def _on_restore_action(self, event) -> None:
 
         # sometimes when we are trying to restore pmb can be resyncing, so we need to retry
         try:
-            event.set_results({"restore-status": "restore started"})
             self.charm.unit.status = MaintenanceStatus("restore started/running")
             self._try_to_restore(backup_id)
+            self._success_action_with_info_log(
+                event, action, {"restore-status": "restore started"}
+            )
```

Comment:
Will you report "restore started" after restore completed (could be a long process)?

Reply:
yes, now it will be